new/LIT6583RHMaryam.xml

```diff
@@ -0,0 +1,60 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
```

Comment:
Did some branch confusion happen here? LIT6583RHMaryam was created through https://github.com/BetaMasaheft/Works/pull/764/files as well. If this file here is a redundant doublet, I think the best would be if you deleted it from this branch.

Reply:
Sorry, as usual I don't know how to sort these branch problems... Do you mean that I should switch to branch LIT6585RHPentecost and then delete the file LIT6583RHMaryam from my GitHub folder?

Reply:
Yes, exactly, that was the problem. I did it now quickly, sorry this got lost among other things for so long.